source/chapters/library.rst

```diff
@@ -866,8 +871,42 @@ different location. Select :guilabel:`Use Default Library` to reset.
   Mac partition mounted in Linux, you can load it and use your iTunes tracks and
   playlists as well.
 
-.. seealso:: External libraries can be disabled under
-             :menuselection:`Preferences --> Library`.
+.. _library-serato:
+
+Using the Serato library
+------------------------
+
+Mixxx supports reading your Serato library and crates both from your local hard drive as well as portable USB drives. Smart crates are currently not supported.
+
+Serato stores information such as hot cues, track color and beatgrid in the file tags.
+Regardless of whether a track is loaded into deck from the libray or directly from the file browser, Mixxx will import that information automatically if present.
+
+Saved loops are imported as well, but since Mixxx is lacking support for saved loops, only the first loop is usable in Mixxx.
+All other saved loops can be used like regular hotcues for now.
+   
+Waveforms, Gain values and "Flips" are not imported from Serato.
+
+.. note:: Due to differences in the metadata format, importing Serato information from Ogg Vorbis files is currently not supported.
+
+**How to import Serato libraries from external USB drive**
+
+* Copy music library from Serato into external USB drive
+* Plug in USB drive while Mixxx is running
+* Click on :guilabel:`Serato` in the library. All USB drives containing a Serato library will be detected automatically.
+* For each removable device, the library and all crates from Serato will accessible.
+
+If you have already added a track containing Serato Metadata to your Mixxx library in a Mixxx version that didn't support reading that data, you can reimport it from the file's tags via the track context menu.
+
+.. warning:: This action will clear your existing cuepoints in Mixxx if the file contains Serato Metadata
+
+**To reimport metadata for files**
```

Comment:
```suggestion
To reimport metadata for files:
```

Reply:
+1

Reply:
This also marked as resolved but not fixed.

Reply:
> This also marked as resolved but not fixed.

Fixed now, sorry.